CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### [Unreleased]
 
+### [1.0.2] 2021-01-12
```

Comment:
do we not do diff links for this changelog?

Reply:
I had just copied the style from the other SDK changelogs - we would need to start using tagged releases to support this I think. Do we know if there is a way to do that retroactively?

Reply:
Yea, you should just be able to tag previous commits. Definitely don't need to do this now.

Reply:
Beautiful - will tag this release properly as 1.0.2 so future diff links are less ugly.